src/Core/Content/Product/Aggregate/ProductPrice/ProductPriceCollection.php

```diff
@@ -34,6 +36,18 @@ public function sortByQuantity(): void
         });
     }
 
+    public function sortByPrice(Context $context): void
+    {
+        $this->sort(function (ProductPriceEntity $a, ProductPriceEntity $b) use ($context) {
+            if ($context->getTaxState() === CartPrice::TAX_STATE_GROSS) {
+                return $a->getPrice()->first()->getGross() <=> $b->getPrice()->first()->getGross();
+            }
+
+            return $a->getPrice()->first()->getNet() <=> $b->getPrice()->first()->getNet();
```

Comment:
```suggestion
            return $a->getPrice()->first()->getNet() <=> $b->getPrice()->first()->getNet();
        });    
```
Please remove the trailing line break after return. Otherwise this looks good to me 👍